implementations/CMIP6-UofT/add_CMIP6.py

```diff
@@ -50,15 +164,23 @@ def handle_ingestion_error(self, error: str, item_name: str, item_data: MutableM
 
     def create_stac_item(self, item_name: str, item_data: MutableMapping[str, Any]) -> MutableMapping[str, Any]:
         # TODO: next step is to implement this
-        self.validator(**item_data)
-
-    def validate_stac_item_cv(self, data: MutableMapping[str, Any]) -> bool:
-        # TODO: next step is to implement this
-        pass
 
+        # Create STAC item geometry from CFMetadata
+        item = dict(
+            id = make_cmip6_id(item_data["attributes"]),
+            geometry = ncattrs_to_geometry(item_data["groups"]["CFMetadata"]["attributes"]),
+            bbox = ncattrs_to_bbox(item_data["groups"]["CFMetadata"]["attributes"]),
+            properties = Properties(item_data["attributes"]).dump_model(),
```

Comment:
This is very interesting. I haven't used Pydantic but I can see the convenience.

Reply:
It also allows us to rename the attributes (alias).